integration/service/test/test_cron_use_case_plot.py

```diff
@@ -0,0 +1,67 @@
+import yaml
+import sys
+from glob import glob
+from pandas import DataFrame, to_datetime
+
+report_names = glob('geopm-report-*.yaml')
+report_list = []
+for rn in report_names:
+    with open(rn) as fid:
+        report = yaml.load(fid, Loader=yaml.SafeLoader)
```

Comment:
It would be nice to use CSafeLoader when available since it parses yaml faster than SafeLoader